language-server/modules/langserver-core/src/test/resources/completion/new/class_def/config/config1.json

```diff
@@ -289,10 +289,10 @@
       ]
     },
     {
-      "label": "ballerina/lang.transaction",
```

Comment:
Do we need these config json changes?

Reply:
These are for lang server tests. When the tests failed I regenerated these json files to support the runtime api